<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PromotionSelectoinDialog</class>
 <widget class="QDialog" name="PromotionSelectoinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="rb_Queen">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Queen</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_Rook">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Rook</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_Bishop">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Bishop</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_Knight">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Knight</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_Accept">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
